geniza/corpus/templates/corpus/document_detail.html

```diff
@@ -29,42 +29,49 @@ <h1>
         {# tabs #}
         {% include "corpus/snippets/document_tabs.html" %}
 
-        {# metadata #}
-        <dl class="metadata">
-            <dt class="sr-only">{% translate 'Shelfmark' %}</dt>
-            <dd>{{ document.shelfmark }}</dd>
-            {% if document.doctype %}
-                <dt class="sr-only">{% translate 'Document Type' %}</dt>
-                <dd>{{ document.doctype }}</dd>
+        <section>
+            <h2 class="sr-only">
+                {# Translators: label for metadata section #}
```

Comment:
Not how clear this will be for translators, let's add a little more detail:

```suggestion
                {# Translators: label for document metadata section (editor, date, input date) #}
```